dictionaries/CallMap.php

```diff
@@ -12266,7 +12266,7 @@
 'SoapFault::getPrevious' => ['?Exception|?Throwable'],
 'SoapFault::getTrace' => ['list<array<string,mixed>>'],
 'SoapFault::getTraceAsString' => ['string'],
-'SoapFault::SoapFault' => ['object', 'faultcode'=>'string', 'faultstring'=>'string', 'faultactor='=>'string', 'detail='=>'string', 'faultname='=>'string', 'headerfault='=>'string'],
+'SoapFault::SoapFault' => ['object', 'faultcode'=>'string', 'faultstring'=>'string', 'faultactor='=>'string', 'detail='=>'mixed', 'faultname='=>'string', 'headerfault='=>'string'],
```

Comment:
Changing the details is ok, but the other props also don't seem to match the signature of [SoapFault::__construct()](https://www.php.net/manual/en/soapfault.construct.php)

Reply:
I have updated the constructor signature according to the documentation. And also the property map has been updated.

Reply:
@orklah  : Do the names of these parameters matter much?
Since It is actually:

```php
#[Pure]
public function __construct(
        array|string|null $code,
        string $string, 
        ?string $actor = null,
        mixed $details = null,
        ?string $name = null,
        mixed $headerFault = null
    ) {
```

See https://github.com/php/php-src/blob/master/ext/soap/soap.stub.php#L38

Reply:
Names do matter, for when Psalm checks calls with named arguments.

Reply:
I've updated the signatures for [SoapFault::__construct()](https://github.com/php/php-src/blob/1d54097435bc09a287cb50b6e166eccba19f2925/ext/soap/soap.stub.php#L38) and [SoapFault::SoapFault()](https://github.com/php/php-src/blob/6d5f2ba78d85f24e69eee5c7c3948171b9fc3b05/ext/soap/soap.c#L855-L856).